### PR TITLE
new item modification

### PR DESCRIPTION
**Pull request by itaminul**
ok all right

### CODE REVIEW

**Review by itaminul — COMMENTED** [review]
ok all right

File: src/components/pages/CreateItem.tsx [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    
+}
+
+function CreateItem ({ title, visible, onCancel } : CreateModalProps) {
+    const [form] = Form.useForm();
+    const { data } = useGetSupplierAllDataQuery();
+    const [createItem] = useCreateItemMutation();
+
+    
+    const onFinish = async(values: any) => {
```
Comment (bablukpik) [inline src/components/pages/CreateItem.tsx:20]: Please remove any and use specific type here, you can follow this one https://github.com/bablukpik/inventory-hasura-graphql/blob/main/src/components/Products/AddProduct.tsx#L15C42-L15C42

File: src/components/pages/CreateItem.tsx [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+            manufactureDate: new Date(values.manufactureDate).toISOString(),
+            expireDate: new Date(values.expireDate).toISOString(),
+            taxRate: Number(values.taxRate),
+            reorderLabel: Number(values.reorderLabel),
+            itemImage: values.itemImage,
+            remarks: values.remarks,
+            supplierId: Number(values.supplierId),
+            itemGroupId: Number(values.itemGroupId)
+          };
+          console.log("new Item", newItem);
```
Comment (bablukpik) [inline src/components/pages/CreateItem.tsx:38]: Please remove console log

File: src/components/pages/CreateItem.tsx [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+        } catch (error) {
+          console.error('Error creating item:', error);
+        }
+      };
+
+      const disabledDate = (current: any) => {
+        const today = new Date();
+        return current && current > today;
+      }
+console.log("data", data);
```
Comment (bablukpik) [inline src/components/pages/CreateItem.tsx:59]: Console log

**Pull request closed without merging**